internal/service/ssmcontacts/testdata/tmpl/contact_channel_tags.gtpl

```diff
@@ -0,0 +1,26 @@
+resource "aws_ssmcontacts_contact_channel" "test" {
+{{- template "region" }}
+  contact_id = aws_ssmcontacts_contact.test.arn
+
+  delivery_address {
+    simple_address = "test@example.com"
+  }
+
+  name = var.rName
+  type = "EMAIL"
+
+{{- template "tags" . }}
+}
+
+resource "aws_ssmcontacts_contact" "test" {
+{{- template "region" }}
+  alias = "test-contact-for-${var.rName}"
+  type  = "PERSONAL"
+
+  depends_on = [data.aws_ssmincidents_replication_set.test]
+{{- template "tags" . }}
```

Comment:
Exclude the tags template for any resources which are not the primary subject of the test case.

```suggestion
```